combat-trainer.lic

```diff
@@ -1779,6 +1782,32 @@ class TrainerProcess
       bput(%w[weave bob circle].sample, 'roundtime', 'There is nothing else', 'Face what', 'You must be closer', 'You must be standing', 'Strangely, you don\'t feel like fighting right now', 'flying too high for you to attack') unless game_state.npcs.empty?
     when 'Analyze'
       analyze(game_state, 0)
+    when 'Locks'
+      return if game_state.loaded
+      return if left_hand
+      boxes = get_boxes(@pet_box_source)
+      return if boxes.length == 0
+      box = boxes.first
+      # Check what's in the right hand so we can get it back
+      rhitem = @equipment_manager.item_by_desc(right_hand) if right_hand
+      @equipment_manager.stow_weapon("#{rhitem.adjective} #{rhitem.name}") if rhitem
+      return if right_hand  # Couldn't empty right hand for some reason (typically a summoned weapon).  No picking allowed
```

Comment:
❓ If you check the summoning code, there's some logic to check if your current weapon is summoned. I suggest checking for that, outputting a message, and returning. What do you think?

Reply:
Holding a summoned weapon is one of the things that broke the logic when I was first working on / testing this.  I'm not sure if there are other things that might also get in the way that might not be summoned weapons.

I think breaking the pick execution if there's something unexpected in your hands is a safe guard

Reply:
:+1: